### PR TITLE
CNV-24863: Review known issues and bug fixes

### PR DESCRIPTION
**Pull request by aspauldi**
Version(s): 4.14

Issue: [CNV-24863](https://issues.redhat.com/browse/CNV-24863)

Links to docs preview: 
- [Bug fixes](https://67169--docspreview.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-14-release-notes#virt-4-14-bug-fix)
- [Known issues](https://67169--docspreview.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-14-release-notes#virt-4-14-known-issues)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by aspauldi** [comment]
@tiraboschi or @kbidarkar, Can you review the two Known Issues CNV-22036 (in the **Known issues Nodes** section) and BZ2157951 (in the **Known issues Virtualization** section) in this PR?  They are currently included in the 4.14 release notes--should they be? Thanks!

**Comment by tiraboschi** [comment]
> @tiraboschi or @kbidarkar, Can you review the Known Issues [CNV-22036](https://issues.redhat.com//browse/CNV-22036) (in the **Known issues Nodes** section) 

Nothing got really implemented about CNV-22036 so yes, this should stay here.

> and BZ2157951 (in the **Known issues Virtualization** section) in this PR? They are currently included in the 4.14 release notes--should they be? Thanks!

This is not new nor specific for 4.14 customers that were eventually already hitting that on 4.13.
Although it could still be an issue.
Not so sure.

File: virt/release_notes/virt-4-14-release-notes.adoc [hunk 1]
```diff
@@ -252,68 +252,75 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
...
-//4.14 to be added by Pan
-//CNV-34198 (BZ 2237287)
-
-
+[discrete]
+[id="virt-4-14-ki-webconsole"]
+==== Web console
+//CNV-34771 from Ying
+* If you upgrade from OpenShift Container Platform 4.13 to 4.14, some pages in the web console might close unexpectedly. (link:https://issues.redhat.com/browse/CNV-34771[*CNV-34771*])
+** As a workaround, upgrade to {VirtProductName} 4.14.
```
[the remark this answers is not in the document]
Comment (aspauldi) [reply virt/release_notes/virt-4-14-release-notes.adoc]: Avital edited her comment ^ to this:

If you upgrade OpenShift Container Platform 4.13 to 4.14 without upgrading OpenShift Virtualization, the Virtualization pages of the web console crash. You must upgrade the OpenShift Virtualization Operator to 4.14 manually or set your subscription approval strategy to "Automatic."

File: virt/release_notes/virt-4-14-release-notes.adoc [hunk 2]
```diff
@@ -252,67 +252,76 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
...
+//4.14 Leave in per Adam and Stu; will be doc'd in 4.15
+* If you clone more than 100 VMs using the `csi-clone` cloning strategy, then the Ceph CSI might not purge the clones. Manually deleting the clones might also fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055595[*BZ#2055595*])
+** As a workaround, you can restart the `ceph-mgr` to purge the VM clones.
 
-//4.14 Leave in per Stu (fix deferring to 4.15)
-* The Pod Disruption Budget (PDB) prevents pod disruptions for migratable virtual machine images. If the PDB detects pod disruption, then `openshift-monitoring` sends a `PodDisruptionBudgetAtLimit` alert every 60 minutes for virtual machine images that use the `LiveMigrate` eviction strategy. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2026733[*BZ#2026733*])
-** As a workaround, xref:../../monitoring/managing-alerts.adoc#silencing-alerts_managing-alerts[silence alerts].
+//CNV-34198 (BZ 2237287)
+* If you use Portworx as your storage solution on AWS and create a VM disk image, the created image might be smaller than expected due to the filesystem overhead being accounted for twice. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2237287[*BZ#2237287*])
```
Comment (mjpytlak) [inline virt/release_notes/virt-4-14-release-notes.adoc:294]:
```suggestion
* If you use Portworx as your storage solution on AWS and create a VM disk image, the image might be smaller than expected due to the filesystem overhead being accounted for twice. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2237287[*BZ#2237287*])
```

File: virt/release_notes/virt-4-14-release-notes.adoc [hunk 3]
```diff
@@ -252,67 +252,76 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
...
 
-//CNV-24889 (Avital mentions possible RN in CNV-32472)
-
-//Known issues from 4.13 (and several carried over from 4.12) begin here.Leave in all for now; will be re-reviewed for 4.14.
+[discrete]
+[id="virt-4-14-ki-monitoring"]
+==== Monitoring
+//4.14 Leave in per Stu (fix deferring to 4.15)
+* The Pod Disruption Budget (PDB) prevents pod disruptions for migratable virtual machine images. If the PDB detects pod disruption, then `openshift-monitoring` sends a `PodDisruptionBudgetAtLimit` alert every 60 minutes for virtual machine images that use the `LiveMigrate` eviction strategy. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2026733[*BZ#2026733*])
+** As a workaround, xref:../../monitoring/managing-alerts.adoc#silencing-alerts_managing-alerts[silence alerts].
```
Comment (mjpytlak) [inline virt/release_notes/virt-4-14-release-notes.adoc:260]: Not a now thing as I see this is existing formatting, but using a nested bullet for the workaround seems odd. Curious as to why a list continue (`+`) is not used. Perhaps a global formatting fix after you GA.

**Review by mjpytlak — COMMENTED** [review]
@aspauldi One small update. Otherwise lgtm.

File: virt/release_notes/virt-4-14-release-notes.adoc (hunk 3 above)
Comment (aspauldi) [reply virt/release_notes/virt-4-14-release-notes.adoc:260]: @ousleyp, okay with you if I change the formatting as suggested?

**Review by apinnick — APPROVED** [review]

**Pull request merged**